utils/fish/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2017 Shane Peelar
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+BASE_VERSION:=2.5.0
+
+PKG_NAME:=fish
+PKG_VERSION:=$(BASE_VERSION)
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(BASE_VERSION).tar.gz
+PKG_SOURCE_URL:=https://fishshell.com/files/$(BASE_VERSION)/
+PKG_SHA1SUM:=c5981245b45ea61b765008299b5ac477657089c3
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BASE_VERSION)
+
+PKG_LICENSE:=GPL-3.0+
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Shane Peelar <lookatyouhacker@gmail.com>
+
+PKG_FIXUP:=autoreconf
+
+EXTRA_CXXFLAGS+= -std=gnu++11
```

Comment:
This definition needs to be moved to occur after including package.mk (like with the rest of the compile/linker options)

Please test your builds before submitting them.